peachjam/js/peachjam.ts

```diff
@@ -137,6 +138,20 @@ class PeachJam {
     });
   }
 
+  setupPopovers () {
+    document.querySelectorAll('[data-bs-toggle="help-popover"]').forEach((el) => {
+      // @ts-ignore
+      new window.bootstrap.Popover(el, {
+        html: true,
+        content: `
+        ${el.getAttribute('data-bs-content')}
+         <div><a href="${el.getAttribute('href')?.split('#')[1] || '#'}" target='_blank' rel='noopener noreferrer'>Learn more</a></div>
```

Comment:
The goal is to open the docs page in a new browser tab, and the popup can close. So I think the default behaviour is sufficient, unless I'm missing something.

Reply:
@longhotsummer With the way it was, clicking on the help button triggered the browser to navigate away immediately because its technically an <a> anchor tag with a href attribute